packages/browser-wallet/src/types/react-i18next.d.ts

```diff
@@ -0,0 +1,15 @@
+// import the original type declarations
+import 'react-i18next';
+
+import type { resources, defaultNS } from '../popup/shell/i18n';
+
+// react-i18next versions higher than 11.11.0
+declare module 'react-i18next' {
```

Comment:
Why do you need to add this?
Does the package not have proper ts support? :thinking:

Reply:
They do, this is a way to extend their types to include the specific translations for the project. It's how they documented how to do this on their website. Without this, selecting a translation using any method will not be strongly typed, and it is when this is included.